.github/workflows/publish_site.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
nit, but we already have https://github.com/aep-dev/aep.dev/blob/main/.github/workflows/publish.yaml. Is it worth joining this workflow with that one?

Reply:
I'd like to keep them separate so they kick off separate runners. We don't want one runner to interrupt another.